### PR TITLE
Revert "fixtodo:validate events on PVCs in integration volume binding test"

### LINKED ISSUE #63302: TestVolumeBindingStress flakes
Seeing flakes in the TestVolumeBindingStress test:

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/58474/pull-kubernetes-integration/9177/

The recent changes to this test in https://github.com/kubernetes/kubernetes/pull/62929 assume events are a reliable message bus, which they are not.

### PR DESCRIPTION
**Pull request by liggitt**
This reverts commit 910fd1953edfb10828ad7cccdea58cc573519352.

Events are not a reliable message bus... they throttle, deduplicate, and drop events over fairly conservative rate limit targets.

This commit caused test flakes, so reverting.

Fixes #63302

```release-note
NONE
```

### CODE REVIEW

File: test/integration/scheduler/volume_binding_test.go [hunk 1]
```diff
@@ -398,7 +386,7 @@ func setupCluster(t *testing.T, nsName string, numberOfNodes int) *testConfig {
 		ClaimInformer:             informers.Core().V1().PersistentVolumeClaims(),
 		ClassInformer:             informers.Storage().V1().StorageClasses(),
 		PodInformer:               informers.Core().V1().Pods(),
-		EventRecorder:             eventRecorder,
+		EventRecorder:             nil, // TODO: add one so we can test PV events
```
Comment (msau42) [inline test/integration/scheduler/volume_binding_test.go]: Can you remove this TODO

Comment (liggitt) [reply test/integration/scheduler/volume_binding_test.go]: done

**Comment by hanxiaoshuai** [comment]
@liggitt @msau42  Oh, sorry for the delay. I was on labour holiday. Thanks for your revert, liggit

**Pull request merged**